crates/router/tests/connectors/adyen.rs

```diff
@@ -453,6 +460,127 @@ async fn should_fail_capture_for_invalid_payment() {
     );
 }
 
+/******************** Payouts test cases ********************/
+// Create SEPA payout
+#[cfg(feature = "payouts")]
+#[actix_web::test]
+async fn should_create_sepa_payout() {
+    let mut payment_info = AdyenTest::get_payment_info();
```

Comment:
add all your default data for payout within get_payout_info and use it in all other places below
```suggestion
    let mut payout_info = AdyenTest::get_payout_info();
```